contracts/axelar-gas-service/tests/test.rs

```diff
@@ -34,6 +46,14 @@ fn message_id(env: &Env) -> String {
     )
 }
 
+fn dummy_destination_data(env: &Env) -> (String, String) {
+    let destination_chain: String = String::from_str(&env, "ethereum");
+    let destination_address: String =
+        String::from_str(&env, "0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59");
```

Comment:
perhaps
```suggestion
    let destination_address: Address = Address::generate(&env);
```
and `to_string()` can be called

Reply:
Haha, you're too quick! Working on it